pytorch_pipeline/components/trainer/component.py

```diff
@@ -61,13 +80,11 @@ def __init__(
             if not data_module_class:
                 raise ValueError(f"Unable to load data_module_file - {data_module_file}")
 
-            self.ptl_trainer = Executor().Do(
-                model_class=model_class,
-                data_module_class=data_module_class,
-                data_module_args=data_module_args,
-                module_file_args=module_file_args,
-                trainer_args=trainer_args,
-            )
+            # self.ptl_trainer = Executor().Do(
```

Comment:
working on executor restructuring. will enable this once the restructuring is done.